files/free-jqgrid/update.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packageManager": "github",
+  "name": "free-jqgrid",
+  "repo": "free-jqgrid/jqGrid",
+  "files": {
+    "basePath": "/",
+    "include": [
+      "css/*",
```

Comment:
You need to use `./` if you want to retain the directory structure.
For example `./css/*` `./js/jquery.jqgrid.min.js`